website/pages/en/developing/creating-a-subgraph/install-the-cli.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Install the Graph CLI
+---
+
+> In order to use your subgraph on The Graph's decentralized network, you will need to [create an API key](/deploying/subgraph-studio-faqs/#2-how-do-i-create-an-api-key) in [Subgraph Studio](https://thegraph.com/studio/apikeys/). It is recommended that you add signal to your subgraph with at least 3,000 GRT to attract 2-3 Indexers. To learn more about signaling, check out [curating](/network/curating/).
+
+## Overview
+
+The [Graph CLI](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli) is a command-line interface that facilitates developers' commands for The Graph. It processes a [subgraph manifest](/creating-a-subgraph/subgraph-manifest/) and compiles the [mappings](/creating-a-subgraph/assemblyscript-mappings/) to create a ready-to-use version of the subgraph.
+
+You need The Graph CLI to build and deploy a subgraph.
+
+## Getting Started
+
+### Install the Graph CLI
+
+The Graph CLI is written in TypeScript, and you must have `node` and either `npm` or `yarn` installed to use it. Check for the [most recent](https://github.com/graphprotocol/graph-tooling/releases?q=%40graphprotocol%2Fgraph-cli&expanded=true) CLI version.
```

Comment:
This is something that devs with more Javascript experience should weigh in on: we should tell people how to install these things, incuding actual commands to run, for popular platforms, or link to the instructions upstream. As somebody who has little JS experience, I always found these steps very annoying and intimidating.

I've also been using `pnpm` instead of `yarn` lately, and it seems nicer/faster. We should have a set of tools that we recommend and where we have good instructions for how to install them, and just mention the other ones.

But all this needs input from people who actually know what they are doing with JavaScript (not me)